src/app/containers/ArticleMain/index.jsx

```diff
@@ -22,17 +26,50 @@ const componentsToRender = {
 };
 
 const ArticleMain = ({ articleData }) => {
-  const { content, metadata, promo } = articleData;
-  const { blocks } = content.model;
+  const {
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+    articleAuthor,
+  } = useContext(ServiceContext);
+
+  const processedData = processArticleData({
```

Comment:
`processArticleData` is poorly named imo. Process it how? for what?

Reply:
Yeh I couldn't think of a better name. Any suggestions? It takes the API response data and transform it into props for components.